controllers/chat/moveToAnon.js

```diff
@@ -105,6 +110,11 @@ const moveToAnon = async (req, res) => {
             newStateMemberWithAnonInfo[member].anon_user_info_emoji_name =
               new_anon_user_info_emoji_name;
 
+            newStateMemberWithAnonInfo[member].user.name =
+              new_anon_user_info_color_name + ' ' + new_anon_user_info_emoji_name;
+            newStateMemberWithAnonInfo[member].user.username =
+              new_anon_user_info_color_name + ' ' + new_anon_user_info_emoji_name;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, consider avoiding direct mutation of the `user` object's `name` and `username` properties. Instead, create a new object with the updated properties.

```diff
-            newStateMemberWithAnonInfo[member].user.name =
-              new_anon_user_info_color_name + ' ' + new_anon_user_info_emoji_name;
-            newStateMemberWithAnonInfo[member].user.username =
-              new_anon_user_info_color_name + ' ' + new_anon_user_info_emoji_name;

+            newStateMemberWithAnonInfo[member].user = {
+              ...newStateMemberWithAnonInfo[member].user,
+              name: new_anon_user_info_color_name + ' ' + new_anon_user_info_emoji_name,
+              username: new_anon_user_info_color_name + ' ' + new_anon_user_info_emoji_name,
+            };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->